docs/user/flyout-menu.rst

```diff
@@ -0,0 +1,121 @@
+Flyout Menu
+===========
+
+When you are using a Read the Docs site,
+you will likely notice that we embed a menu on all the documentation pages we serve.
+This is a way to expose the functionality of Read the Docs on the page,
+without having to have the documentation theme integrate it directly.
+
+Functionality
+-------------
+
+The flyout menu provides access to the following bits of Read the Docs functionality:
+
+* A :doc:`version switcher </versions>` that shows users all of the active, unhidden versions they have access to.
+* :doc:`Downloadable formats </downloadable-documentation>` for the current version, including HTML & PDF downloads that are enabled by the project.
+* Links to the Read the Docs dashboard for the project.
+* Links to your :doc:`VCS provider </integrations>` that allow the user to quickly find the exact file that the documentation was rendered from.
+* A search bar that gives users access to our :doc:`/server-side-search` of the current version.
+
+Closed
+~~~~~~
+
+.. figure:: /_static/images/flyout-closed.png
+
+    The flyout when it's closed
+
+Open
+~~~~
+
+.. figure:: /_static/images/flyout-open.png
+
+    The opened flyout
+
+Information for theme authors
+-----------------------------
+
+People who are making custom documentation themes often want to specify where the flyout is injected,
+and also what it looks like.
+We support both of these use cases for themes.
+
+Defining where the flyout menu is injected
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The flyout menu injection follows this logic:
+
+* When a ``<div id="readthedocs-embed-placement">`` is defined, inject the flyout menu there.
+* If a page is an old version of the Read the Docs Sphinx theme, update the place where the theme expected it to be injected.
```

Comment:
This bullet is not clear to me. What does "update the place where the theme expected it to be injected" mean?

Reply:
We could probably just remove this. It's an internal detail most users don't need to care about.